.gitattributes

```diff
@@ -0,0 +1,15 @@
+tests               export-ignore
+.editorconfig       export-ignore
+.gitattributes      export-ignore
+.gitignore          export-ignore
+.phpcs.xml          export-ignore
+CONTRIBUTING.md     export-ignore
+phpunit.xml         export-ignore
+README.md           export-ignore
```

Comment:
What do you think about allowing `README.md` to be installed along with PHP code?

Reply:
Good call. It never hurts to always distribute the docs! Fixed. :)